dev-util/ctags/ctags-20170704.ebuild

```diff
@@ -3,13 +3,13 @@
 
 EAPI=6
 
-inherit autotools
+inherit autotools vcs-snapshot
 
 COMMITHASH="0645b2c833e4ca956970cc96fab32a1b04c0c55c"
 
 DESCRIPTION="Exuberant Ctags creates tags files for code browsing in editors"
 HOMEPAGE="https://ctags.io/ https://github.com/universal-ctags/ctags"
-SRC_URI="https://github.com/universal-ctags/ctags/archive/${COMMITHASH}.zip -> ${P}.zip"
+SRC_URI="https://github.com/universal-ctags/${PN}/tarball/${COMMITHASH} -> ${P}.tar.gz"
```

Comment:
1. Please do not put `${PN}` in prefix of SRC_URI, since it's not required to always match the package name, and it gives no real advantage.
2. The `tarball/` URL is deprecated, just replace `.zip` with `.tar.gz` instead.